extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -63,14 +65,20 @@ require([
 				height: 337,
 				modelName: 'wiki',
 				opts: {
-					rule_language : window.wgContentLanguage
+					rule_language: window.wgContentLanguage
 				}
 			}
 		};
 
 	function prepareRailRecirculation(options) {
+		var request;
```

Comment:
New line below

Reply:
Not anymore 🔨

Reply:
i'd suggest keeping not defined variables below the defined ones or rewrite that to
```
var isRecirculationABTest = window.Wikia.AbTest.inGroup('RIGHT_RAIL_RECIRCULATION_SOURCE', 'TOPIC_FEED') &&	getCurationCMSTopic();
var request = isRecirculationABTest ? getDataFromCurationCMS() : liftigniter.prepare(options);
```